.vitepress/theme/style.css

```diff
@@ -1,3 +1,8 @@
+iframe {
+  width: 100%;
+  border: 1px solid #CCC;
+  padding: 50px;
```

Comment:
**Suggestion**: To ensure this doesn’t look weird in dark mode.

```suggestion
  padding: 50px;
  background-color: #fff;
```

Reply:
Resolving this following https://github.com/kumahq/kuma-gui/pull/1807#issuecomment-1828302119.